crates/cli/src/bin/wasm-bindgen-test-runner/main.rs

```diff
@@ -152,6 +211,9 @@ fn main() -> anyhow::Result<()> {
         },
         Some(_) => bail!("invalid __wasm_bingen_test_unstable value"),
         None if std::env::var("WASM_BINDGEN_USE_DENO").is_ok() => TestMode::Deno,
+        None if std::env::var("WASM_BINDGEN_USE_BROWSER").is_ok() => TestMode::Browser {
+            no_modules: std::env::var("WASM_BINDGEN_USE_NO_MODULE").is_ok(),
+        },
```

Comment:
Why this change?
I think we could at least discuss and do this in a separate PR.

Reply:
I believe I added it to be able to execute the tests on the browser without having require the runtime in the code, that way you can check if your assembly runs in all runtimes without having to recompile it.

Reply:
This is a great feature, but still belongs in a separate PR.

Reply:
This is the same issue with the deno fixes, I didn't planned for them, both had the same underlying objectives:
- add tests to make sure the application remained working as close as possible to the original, because of the docopt and nextest implied changes
- enforce that the nextest api is implemented consistently across all the supported runtimes

That being said, I actually do agree with you, but please advise me the best way to do this.
If I open PRs for them, once they are approved and merged, should a rebase to the latest version work?

Reply:
I see ... I guess the issue is that running the tests on all targets would be more complicated without this change.
I'm fine with leaving it in this PR in this case, but it needs some added documentation.